### PR TITLE
Added CHANGELOG.md

### PR DESCRIPTION
**Pull request by JohnKallies**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# Announcing the release of OSMT 2.5.0
+February 28, 2023
+## Enhancements
+OSMT 2.5.0 brings 2 new feature enhancements
+
+- Addition of a "My Workspace" feature, allowing a collection-like place for a user to save and sort RSDs. RSDs can be added and removed the the Workspace, and the Workspace can be converted to a real Collection, and exported as a CSV without creating a real Collection. The Workspace can also be reset (emptied).
```
Comment (manuel-delvillar) [inline CHANGELOG.md:6]: We have double "the" _be added and removed the the Workspace_

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+## Enhancements:
+- Completed UI implementation of role-based access in OSMT. See "Role-based Access in OSMT" in README.md
+
+## Patching and Developer Tooling:
+- Adjusted/upgraded OpenCSV and Jackson DataBind dependencies to address critical vulnerabilities
+  - CVE-2022-42889
+  - CVE-2022-42004
+- Added development as default local Angular configuration
+  - The Maven build continues to to a production Angular build, but the npm commands now default to development
```
Comment (manuel-delvillar) [inline CHANGELOG.md:93]: Double "to"

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+February 28, 2023
+## Enhancements
+OSMT 2.5.0 brings 2 new feature enhancements
+
+- Addition of a "My Workspace" feature, allowing a collection-like place for a user to save and sort RSDs. RSDs can be added and removed the the Workspace, and the Workspace can be converted to a real Collection, and exported as a CSV without creating a real Collection. The Workspace can also be reset (emptied).
+
+- Collections now have a "description" field. The description is indexed and searchable.
+
+- An RSD now has a "Copy Public URL" button, to simplify coping the canonical URL for that RSD.
+-
```
Comment (manuel-delvillar) [inline CHANGELOG.md:11]: This is empty

File: CHANGELOG.md (hunk 1 above)
Comment (JohnKallies) [reply CHANGELOG.md:6]: This was copy/paste from the release announcements in GitHub. It's wrong there too. I'll fix it, thank you.

File: CHANGELOG.md (hunk 2 above)
Comment (JohnKallies) [reply CHANGELOG.md:93]: same

File: CHANGELOG.md (hunk 3 above)
Comment (JohnKallies) [reply CHANGELOG.md:11]: same

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+# Announcing the release of OSMT 2.1.0
+Novemeber 4, 2022
+
+This release brings several defect fixes, and completes the feature enhancement for roles within OSMT (admin, curator, viewer).
+
+## Defect fixes:
+- RSD result counts greater than 10,000 are now accurately reports in the UI
+- BLS and O*NET job code hierarchies have been corrected.
+  - This was an import issue, and to benefit from this fix, you will need to reimport the 2018 BLS Job Code metadata, and reindex your ElasticSearch instace. See https://github.com/wgu-opensource/osmt/discussions/251 for more details about using this fix.
```
Comment (jchavez137) [inline CHANGELOG.md:80]: typo 'instace'

Comment (manuel-delvillar) [inline CHANGELOG.md:80]: Typo in "instace"

**Review by manuel-delvillar — APPROVED** [review]

**Pull request merged**